Makefile

```diff
@@ -28,6 +28,7 @@ GO_CMD:=go
 GOFMT_CMD:=gofmt
 KUBECTL_CMD:=kubectl
 BUILDAH_CMD:=buildah
+YAMLLINT_CMD:=yamllint
```

Comment:
Unlike YQ there's no rule to download yamllint if it is not found. Why's that?

Reply:
[yamllint](https://github.com/adrienverge/yamllint) is a python utility. YAMLLINT_CMD follows the same rules as KUBECTL_CMD and BUILDAH_CMD: expects the user to install them in his PATH

Reply:
OK, thanks. Not for this PR, but we should probably think a little bit over how we handle our project dependencies/tools. I started an effort a couple of days ago but got bogged down "in the weeds". But I will probably revisit it soon.